src/components/templates/Audience/Audience.tsx

```diff
@@ -203,21 +203,21 @@ export const Audience: React.FunctionComponent = () => {
   // FIXME: This is really bad, needs to be fixed ASAP
   const partygoersBySeat: WithId<User>[][] = [];
   let seatedPartygoers = 0;
-  partygoers?.forEach((partygoer) => {
+  recentWorldUsers?.forEach((user) => {
```

Comment:
This could probably be replaced by `usePartygoersBySeat`:

- https://github.com/sparkletown/sparkle/blob/staging/src/components/templates/Camp/hooks/usePartygoersBySeat.ts

Reply:
It has this `seatedPartygoers` number used, which is used down in the code. I wouldn't mess with this code in this PR.

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/198